library-app/src/components/Footer/Footer.module.css

```diff
@@ -0,0 +1,50 @@
+.footer {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  min-width: 100%;
+  padding: 0 !important;
```

Comment:
!important is generally bad practice because we don't have control over it. You can achieve the same thing by selecting desired class more specifically. https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#making_selectors_specific_with_and_without_adding_specificity